### ISSUE TITLE
Save only shows an arbitrary subset of folders in home

### ISSUE THREAD

**Issue by php4fan** [bancika/diy-layout-creator#847]
To reproduce on Linux:

- click Ctrl+S or go to File-> Save As
- By default this will open your home folder (`/home/username` e.g. `/home/teo`). If not, navigate to your home folder

Expected behavior
=
I should see all the folders that exist within my home folder, and should be able to open them and navigate to wherever I want to save my project

Observed Behavior
=
It only shows this arbitrary subset of subfolders:
![image](https://github.com/user-attachments/assets/51714437-5423-4fd1-98ca-21793c232df8)

My home directory contains plenty of other non-hidden subfolders, and I actually want to save my project within one of those.

I'm using the Flatpak version on Manjaro Linux

```
Operating System: Manjaro Linux 
KDE Plasma Version: 6.2.4
KDE Frameworks Version: 6.8.0
Qt Version: 6.8.1
Kernel Version: 6.6.54-2-MANJARO (64-bit)
Graphics Platform: Wayland
Processors: 12 × 12th Gen Intel® Core™ i7-1255U
Memory: 15.3 GiB of RAM
Graphics Processor: Mesa Intel® Graphics
Manufacturer: ASUSTeK COMPUTER INC.
Product Name: Vivobook_ASUSLaptop X1502ZA_F1502ZA
System Version: 1.0
```

**Comment by php4fan** [comment]
The non-flatpak version (AUR package "diylc") doesn't have the issue.

This might be a duplicate of https://github.com/bancika/diy-layout-creator/issues/844

**Comment by bancika** [comment]
yeah it's a flatpak issue that doesn't let DIYLC see those folders. I don't know enough about flatpack to tell what exactly causes it. I suggest installing flatseal and trying to manage flatpack permissions from there

https://flathub.org/apps/com.github.tchx84.Flatseal

**Comment by bancika** [comment]
duplicate of #844
